cpp/src/slice2cpp/Gen.cpp

```diff
@@ -1221,7 +1226,12 @@ Slice::Gen::ForwardDeclVisitor::visitConst(const ConstPtr& p)
       << typeToString(p->type(), false, scope, p->typeMetadata(), _useWstring) << " " << fixKwd(p->name()) << " "
       << getDeprecatedAttribute(p) << "= ";
     writeConstantValue(H, p->type(), p->valueType(), p->value(), _useWstring, p->typeMetadata(), scope);
-    H << ';' << sp;
+    H << ';';
+    if (!isConstexprType(p->type()))
```

Comment:
I would add a comment here, not clear what is this about.

Reply:
Added comment.